R/ExpandingMplusItemNames.R

```diff
@@ -0,0 +1,194 @@
+###     test variable names           ###
+### test0     F1-F10...               ###  Pass
+### test1     F1_1-F1_10...           ###  Pass
+### test2     F1_1_1-F1_10...         ###  Pass
+### test3     F1_1-F1_7,F26,F27,F28...###  Pass
+### test4     Fa-Fj...                ###  Pass
+### test5     F_a-F_j...              ###  Pass
+### test6     F_1_a-F_1_j...          ###  Pass
+### test7     Faa-Faj                 ###  Pass
+
+
+
+
+Groups <- unique(fit$parameters$unstandardized$Group)
+### Read dataset in the targetfile
+TargetFile <- {
+  if (grepl("\\.dat$",fit$input$data$file) == T){

```

Comment:
Don't use == T for two reasons:
1) TRUE is a protected word, T is not. Users can create a variable called 'T', and your function will fail.
2) It is superfluous; if grepl("\\.dat$",fit$input$data$file) is true, then grepl("\\.dat$",fit$input$data$file) == T will always be true

Reply:
Thank you for your suggestion and explanation.